### PR TITLE
Inital user docs for video player AB#17082

### PR DESCRIPTION
**Pull request by tmiller-msft**
- Initial user docs for video player

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,9 @@ VoTT helps facilitate an end-to-end machine learning pipeline:
 <!-- command: `markdown-toc -i README.md`-->
 
 <!-- toc -->
+- [VoTT (Visual Object Tagging Tool)](#vott-visual-object-tagging-tool)
```
Comment (tmiller-msft) [inline README.md]: Hmm.. I didn't add this, why did it show up here?

Comment (pjlittle) [reply README.md]: i had a similar problem... some extension I was using in VSCode understands the auto-generated `toc` comments and when I save the markdown, it re-gens the toc.  I usually end up stripping it out, using another editor.

Not sure if anyone else is seeing similar behavior. annoying enough that maybe we should just leave it in.

**Comment by pjlittle** [comment]
LGTM - I like the annotations in the image.

File: README.md [hunk 2]
```diff
@@ -145,7 +148,20 @@ As desired, repeat this process for any additional assets.
...
 
-Coming soon!
+Labeling a video is much like labeling a series of images. When you select a video from the left, it will begin automatically playing, and you have several controls on the player, as seen here:
+![alt text](docs/images/video-player.png "Video Player")
+
+Besides the normal video playback controls, you will see two extra pairs of buttons. 
+
+On the left, you will see the previous and next frame buttons. Clicking these will pause the video, and move to the next appropriate frame as determined by your project settings. For example, if your project settings have a frame extraction rate of 1, these buttons will be moved back or forward 1 second, where if your frame extraction rate was 10, you will be moved back or forward a tenth of a second.
+
+On the right, you will see the previous and next tagged frame buttons. Clicking these will pause the video and move to the next or previous frame that has a tagged region on it, if it exists. If no tagged frame exists to navigate to, it will pause the video and do nothing else.
```
Comment (luisamiranda) [inline README.md]: "On the right are the previous and next tagged frame buttons."
"...move to the next or previous frame that has a previously tagged region on it, if a tagged frame exists."

File: README.md [hunk 3]
```diff
@@ -145,7 +148,20 @@ As desired, repeat this process for any additional assets.
...
+
+You will also notice colored lines along the video's timeline bar. These are the video frames that you have visited and tagged, where a yellow line denotes a frame you've visited and a green line denotes a frame that has tags on it. You may click these bars to quickly navigate to a particular frame as well.
```
Comment (luisamiranda) [inline README.md]: "Colored lines will become visible along the video timeline bar."
"These indicate the video frames that have already been visited."
"A yellow line denotes a frame that has been visited only, while a green line denotes a frame that has been visited and tagged. The bars can be clicked for quick navigation to the indicated frame."

File: README.md [hunk 4]
```diff
@@ -145,7 +148,20 @@ As desired, repeat this process for any additional assets.
...
+
+You can also scrub through the video manually using the timeline to any arbitrary point along the timeline. Note, that the project settings for frame extraction rate are always obeyed, and pausing the video will move to the closest frame according to this project setting. This allows you to set a very low frame extraction rate (say 1 frame per second) for sections of the video you know have little useful items to tag, but raise to a much higher frame extraction rate (say 30 frames per second) to allow fine grain control and tag every frame.
```
Comment (luisamiranda) [inline README.md]: "The timeline can also be used to manually scrub through the video to an arbitrary point, though the project settings for frame extraction rate are always obeyed."
"Pausing the video will move to the closest frame according to this project setting."
"This way, a very low frame rate, such as 1 frame per second, can be set for sections of video known to contain few taggable items, and a much higher frame rate, such as 30 frames per second, to allow fine-grained control."

File: README.md [hunk 5]
```diff
@@ -145,7 +148,20 @@ As desired, repeat this process for any additional assets.
...
+
+While the video is playing, you cannot draw any regions or assign any tags.
```
Comment (luisamiranda) [inline README.md]: "Tagging is not possible while video is playing."

File: README.md [hunk 6]
```diff
@@ -145,7 +148,20 @@ As desired, repeat this process for any additional assets.
 
 ## Labeling a Video
 
-Coming soon!
+Labeling a video is much like labeling a series of images. When you select a video from the left, it will begin automatically playing, and you have several controls on the player, as seen here:
+![alt text](docs/images/video-player.png "Video Player")
+
+Besides the normal video playback controls, you will see two extra pairs of buttons. 
+
+On the left, you will see the previous and next frame buttons. Clicking these will pause the video, and move to the next appropriate frame as determined by your project settings. For example, if your project settings have a frame extraction rate of 1, these buttons will be moved back or forward 1 second, where if your frame extraction rate was 10, you will be moved back or forward a tenth of a second.
```
[the remark this answers is not in the document]
Comment (tmiller-msft) [reply README.md]: Good call on removing the various flavors of you/your. I removed them all (including one you didn't call out) and updated PR.

File: README.md [hunk 7]
```diff
@@ -145,7 +148,20 @@ As desired, repeat this process for any additional assets.
 
 ## Labeling a Video
 
-Coming soon!
+Labeling a video is much like labeling a series of images. When a video is selected from the left, it will begin automatically playing, and there are several controls on the player, as seen here:
+![alt text](docs/images/video-player.png "Video Player")
+
+In addition to the normal video playback controls, there are two extra pairs of buttons. 
+
+On the left, there are the previous and next frame buttons. Clicking these will pause the video, and move to the next appropriate frame as determined by the project settings. For example, if the project settings have a frame extraction rate of 1, these buttons will cause the video to be moved back or forward 1 second, where if the frame extraction rate was 10, the video will be moved back or forward a tenth of a second.
```
Comment (luisamiranda) [inline README.md]: 3rd sentence: "…,while if the frame extraction rate is 10, …"

File: README.md [hunk 8]
```diff
@@ -145,7 +148,20 @@ As desired, repeat this process for any additional assets.
...
+
+On the right, there are the previous and next tagged frame buttons. Clicking these will pause the video and move to the next or previous frame that has a previously tagged region on it, if a tagged frame exists.
+
+Colored lines will also be visible along the video's timeline. These indicate the video frames that have been already been visited. A yellow line denotes a frame that has been visited only, while a green line denotes a frame that has been both visited and tagged. The bars can be clicked for quick navigation to the indicated frame.
```
Comment (luisamiranda) [inline README.md]: 2nd sentence, typo: "...frames that have already been visited"
4th sentence: "The colored lines can be clicked..."
It sounds repetitive to keep saying colored lines, but I think it's clearer.

File: README.md [hunk 9]
```diff
@@ -145,7 +148,20 @@ As desired, repeat this process for any additional assets.
...
+
+The timeline can also be used to manually scrub through the video to an arbitrary point, though the project settings for frame extraction rate are always obeyed. Pausing the video will move to the closest frame according to this project setting. This way, a very low frame extraction rate (say 1 frame per second) can be set for sections of the video known to contain few taggable items, and a much higher frame extraction rate, such as  30 frames per second, to allow fine-grained control.
```
Comment (luisamiranda) [inline README.md]: 3rd sentence: "such as 1 frame per second"
Sorry to be so pedantic.

**Pull request merged**